app/process/roads_bikelanes/roads_bikelanes.lua

```diff
@@ -45,7 +45,7 @@ local excludedRoadsTable = osm2pgsql.define_table({
 function osm2pgsql.process_way(object)
   local tags = object.tags
 
-  ConvertCyclewayOppositeSchema(tags)
```

Comment:
we can move this down and have guards first, right?

Reply:
yes, as long as these guards don't depend on that conversion